### ISSUE TITLE
For MySQL the method IsLocalizationTable should also filter by current database

### ISSUE THREAD

**Issue by okli23** [RickStrahl/Westwind.Globalization#44]
On MySQL, running the query:
```sql
SELECT * FROM INFORMATION_SCHEMA.TABLES where TABLE_NAME=@0
```
returns tables from all databases, that exist on the server.
It means, that additional condition with database name should be added:
```sql
SELECT * FROM INFORMATION_SCHEMA.TABLES where TABLE_NAME=@0
AND TABLE_SCHEMA = @1
```
This change should be performed for MySQL only, as MySQL returns in this column name of database, and MSSQL returns schema name ('dbo' by default).
But, anyway, good job with changing the way of how the IsLocalizationTable works!

**Comment by RickStrahl** [comment]
Problem is we don't know currently what the database is currently... we only have the connection string and aren't specifying schema.

**Comment by okli23** [comment]
For MySQL I would try
```sql
SHOW TABLES LIKE 'Localizations';
```
It returns only tables from current DB, see https://dev.mysql.com/doc/refman/5.0/en/show-tables.html ("SHOW TABLES lists the non-TEMPORARY tables in a *given* database.").

**Comment by RickStrahl** [comment]
Thanks @okli23. Not so familiar with MySql :-)

**Comment by RickStrahl** [comment]
Fixed in 75c9e67b28f8a2fed6169a610e7262d1947f261b.
